### PR TITLE
Add Render as a deployment option

### PR DESCRIPTION
**Pull request by davidmauskop**
I added docs on deploying Strapi to Render, as discussed with @derrickmehaffy and @malgamves: https://github.com/strapi/community-content/issues/166. Feedback is very welcome.

Would it be possible to add a "Deploy on Render" button at the top of the main README? I would've done it myself, but it looks like the button images need to be generated and uploaded to assets.strapi.io.

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
Hi @davidmauskop I've pinged a few people about the button but will review the docs in the meantime :) I don't think it will be a problem though. Thank you for your work!

**Comment by davidmauskop** [comment]
Thanks @derrickmehaffy!

File: docs/v3.x/deployment/render.md
Comment (davidmauskop) [reply docs/v3.x/deployment/render.md:126]: Good idea! We can do:

```
- key: DATABASE_URL
  fromDatabase:
    name: strapi
    property: connectionString
```

And then parse `DATABASE_URL` in `config/env/production/database.js` as in the example you linked to. I'll update our repos and the doc.

File: docs/v3.x/installation/render.md
Comment (davidmauskop) [reply docs/v3.x/installation/render.md:27]: I can help with this, but should point out that a Deploy to Render button here would have to use one of our render-examples repos. If people fork the repo and then click the button in the README, Render will use their fork (it looks at the referrer) and they'll be able to deploy their own changes.

Maybe I should change it to:
```
Click the **Deploy to Render** button in your forked repo's `README` file.
```

**Comment by davidmauskop** [comment]
Thanks! I really appreciate the thorough review.

File: docs/v3.x/installation/render.md
Comment (derrickmehaffy) [reply docs/v3.x/installation/render.md:27]: That might be a better option yeah. I'll check with our marketing team to see about getting a button but if you want to use a placeholder for now thats fine.

Comment (davidmauskop) [reply docs/v3.x/installation/render.md:27]: Another option would be to have a **Deploy to Render** button that deploys Strapi in development mode. People would be able to experiment with creating their own content types, but would lose their changes when they re-deploy (because the new source files won't be committed). We'd emphasize in the doc that this is just for experimenting, not for production apps. What do you think?

Comment (derrickmehaffy) [reply docs/v3.x/installation/render.md:27]: I'd suggest shying away from that generally, there is still a bit of confusion for some of our users about certain PaaS options (Heroku/GAE) about the dynamic nature of the deployment.

I think it's good to introduce these users into a proper source control development flow :)

**Review by meganelacheny — CHANGES_REQUESTED** [review]
Hello! Thank you for your contribution, great work! 👏

I suggested some improvements (mostly to soften sentences that had a promotional tone), and fixed typos.

**Comment by derrickmehaffy** [comment]
@davidmauskop I have the button for our readme, did you have a specific URL you wanted for it?

deploy_on_render

**Comment by davidmauskop** [comment]
Thanks for the great edits @meganelacheny! 

@derrickmehaffy, thanks for the **Deploy on Render** button! I was thinking we could link it to https://render.com/docs/deploy-strapi.

I'll incorporate all of these spot on suggestions and push an update very soon, probably later today (US west coast time).

File: docs/v3.x/installation/render.md
Comment (davidmauskop) [reply docs/v3.x/installation/render.md]: I updated Step 4 to focus more on introducing a source control development flow.

**Comment by davidmauskop** [comment]
I updated the PR to incorporate all the feedback. Let me know if I missed anything!

**Review by derrickmehaffy — APPROVED** [review]
LGTM

**Comment by derrickmehaffy** [comment]
@meganelacheny I'll let you give the final review before merging

**Review by meganelacheny — APPROVED** [review]

**Comment by derrickmehaffy** [comment]
Docs updated :) (The repo readme will be updated when we push a new release)

https://strapi.io/documentation/v3.x/installation/render.html
https://strapi.io/documentation/v3.x/deployment/render.html

**Comment by davidmauskop** [comment]
Thanks for your help @derrickmehaffy and @meganelacheny!

**Comment by derrickmehaffy** [comment]
> Thanks for your help @derrickmehaffy and @meganelacheny!

Thank you for helping make Strapi easy to deploy!

**Pull request merged**
